index.html

```diff
@@ -55,6 +55,15 @@
     <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
     <script src="https://d3js.org/d3.v5.min.js"></script>
     <script src="src/entry-point-client/index.tsx"></script>
+
+    <!--
+      Redirect if src/entry-point-client/index.tsx doesn't load properly.
+      The variable `window.appLoaded` is set at the bottom of index.tsx.
+     -->
+    <script>
+      if (location.pathname === "/" && !window.appLoaded)
```

Comment:
If the synchronous part of the app JS crashes for any reason, this would show the "unsupported browser" page. There would be some false positives in case we just write buggy code and it crashes the app -- but at least they'll be sent to a link where they can contact us.

If we want to be less aggressive in redirecting, we can:
- Only test for the syntax feature we know we need: `try { eval("()=>1") } catch { redirect }`
- Place setting `window.appLoaded` right after the imports in `index.tsx`. Then, if synchronous part of the React code causes any issues, we won't redirect (but of course the user will be left with a broken page).

Let me know what you think!

Reply:
Yeah I think this is great, but I'm going to revise the copy to give us a bit of a messaging out if this is seen in error. Thanks!